data/json/recipes/recipe_weapon.json

```diff
@@ -852,6 +852,35 @@
     "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 } ],
     "components": [ [ [ "pipe", 2 ] ], [ [ "scrap", 3 ] ], [ [ "2x4", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "lever_shotgun",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "mechanics",
+    "skills_required": [ "gun", 3 ],
+    "difficulty": 7,
+    "time": "3 h",
+    "reversible": true,
+    "autolearn": true,
+    "qualities": [
+      { "id": "SAW_M_FINE", "level": 1 },
+      { "id": "SCREW_FINE", "level": 1 },
+      { "id": "WRENCH_FINE", "level": 1 },
+      { "id": "GLARE", "level": 2 }
+    ],
+    "tools": [
+      [
+        [ "oxy_torch", 30 ],
+        [ "welder", 150 ],
+        [ "large_repairkit", 75 ],
+        [ "small_repairkit", 150 ],
+        [ "welder_crude", 225 ],
+        [ "toolset", 225 ]
```

Comment:
This should be a requirement, if it isn't already

Reply:
I copied it from a recipe that's already around, I'll look it up and clean up that one too while I'm at it.